lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -400,7 +400,13 @@ private static Domain getDomain(
     }
 
     @VisibleForTesting
-    public Domain getDomain(Type type, long rowCount, ColumnIndex columnIndex, ParquetDataSourceId id, RichColumnDescriptor descriptor)
+    public static Domain getDomain(
+            Type type,
+            long rowCount,
+            ColumnIndex columnIndex,
+            ParquetDataSourceId id,
+            RichColumnDescriptor descriptor,
+            DateTimeZone timeZone)
```

Comment:
Where is timezone used?

Reply:
It's being used by createTimestampEncoder in the `private static Domain getDomain` method

Reply:
I don’t see anything in this PR using it, so why did it get added here? What am I missing?

Reply:
In the first commit I'm just doing a bit of refactoring to make this method static which required passing in timeZone for https://github.com/trinodb/trino/pull/9945/commits/89133a139afc9703b32197ce8e00e44859aa3379#diff-ad2cef297ca9a945f4efa17ad1e9b36ec97c06578a87eb3b3afaf66eb89c4f16R449
This makes it easier to write UT in the 2nd commit

Reply:
Got it. I was looking at the two commits together and was getting confused.